### PR TITLE
Revert "error dialogs + sound effects"

### PR DESCRIPTION
**Pull request by AlhaririAnas**
I started a review the changes of last pull request already.. 

Please take some time to review the changes before directly merging the pull request into the main branch. Thank you! :)

grafik

### CODE REVIEW

**Comment by Jonah-gr** [comment]
So should I merge your pull request and restore my old one?

**Review by Jonah-gr — APPROVED** [review]

**Comment by AlhaririAnas** [comment]
I've carefully reviewed your code and everything is clear, except for a small adjustment that I've already edited and committed. 

Since I'm not too familiar with such situations, I'll go ahead and merge this pull request. If your changes don't remain, please consider submitting another pull request. 

In general, as always, great job, @Jonah-gr . Thanks! :)

**Comment by heyitsalina** [comment]
sorry! i didn't notice the errors and should have waited for another review before merging 😬

**Pull request merged**
